TypeCobol.LanguageServer/Workspace.cs

```diff
@@ -527,6 +527,9 @@ private void RefreshCustomSymbols()
             try
             {
                 _customSymbols = Tools.APIHelpers.Helpers.LoadIntrinsic(TypeCobolConfiguration.Copies, TypeCobolConfiguration.Format, DiagnosticsErrorEvent); //Refresh Intrinsics
+#if DOMAIN_CHECKER
```

Comment:
Method  Tools.APIHelpers.Helpers.LoadIntrinsic, now contains code:
```
#if DOMAIN_CHECKER
            SymbolTableBuilder.TransfertAllProgramsToIntrinsics();
#endif`
```
So it can be removed here.

Reply:
ok